docs/CONFIG.md

```diff
@@ -96,7 +96,7 @@ URL or configure a TOML-style file in one of the following locations:
 
 Example file:
 
-```toml
+```ini
```

Comment:
Turns out config_resolver uses https://docs.python.org/3/library/configparser.html which uses a simplified INI format which is not actually TOML.  TIL!